src/widgets/Grabber.h

```diff
@@ -46,15 +46,13 @@ class GrabberEvent:public wxCommandEvent
    GrabberEvent(wxEventType type = wxEVT_NULL,
                 wxWindowID winid = 0,
                 const wxPoint& pt = wxDefaultPosition)
-   : wxCommandEvent(type, winid)
+   : wxCommandEvent(type, winid), mPos(pt)
```

Comment:
Just tried this in g++ with -O3 specified:

```c++
class inconstructor
{
public:
   inconstructor(int parm1)
   {
      mParm = parm1;
   }

public:
   int mParm;
};

class ininitializer
{
public:
   ininitializer(int parm1)
   :  mParm(parm1)
   {
   }

public:
   int mParm;
};

inconstructor tryc(123);
ininitializer tryi(456);
```
And here is what the compiler produced:
```gas
   .file "t.cpp"
   .section .text.startup,"x"
   .p2align 4,,15
   .def  _GLOBAL__sub_I_tryc; .scl  3; .type 32;   .endef
   .seh_proc   _GLOBAL__sub_I_tryc
_GLOBAL__sub_I_tryc:
.LFB7:
   .seh_endprologue
   movl  $123, tryc(%rip)
   movl  $456, tryi(%rip)
   ret
   .seh_endproc
   .section .ctors,"w"
   .align 8
   .quad _GLOBAL__sub_I_tryc
   .globl   tryi
   .bss
   .align 4
tryi:
   .space 4
   .globl   tryc
   .align 4
tryc:
   .space 4
   .ident   "GCC: (GNU) 4.9.2"
```
Looks about the same to me...